applications/asset_tracker_v2/sample.yaml

```diff
@@ -4,7 +4,9 @@ tests:
   applications.asset_tracker_v2.nrf_cloud:
     build_only: true
     build_on_all: true
-    platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns thingy91_nrf9160_ns native_sim
+    platform_allow: |
+      nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns nrf9151dk_nrf9151_ns
```

Comment:
Please have each board on a separate line prefixed with a dash. The same applies for every `platform_allow` in this PR.

Reply:
It is done in e.g. https://github.com/nrfconnect/sdk-nrf/blob/main/samples/app_event_manager_profiler_tracer/sample.yaml#L8

Reply:
done, do I get the janitor badge now?